src/referencePackages/Directory.Build.props

```diff
@@ -25,6 +25,7 @@
     <KeyFileDir>$(RepoRoot)src/referencePackages/snk/</KeyFileDir>
     <SignAssembly>true</SignAssembly>
     <PublicSign>true</PublicSign>
+    <FullAssemblySigningSupported Condition="('$(FullAssemblySigningSupported) == '') and ('$(OS)' != 'Windows_NT')">false</FullAssemblySigningSupported>
```

Comment:
```suggestion
    <FullAssemblySigningSupported Condition="('$(FullAssemblySigningSupported)' == '') and ('$(OS)' != 'Windows_NT')">false</FullAssemblySigningSupported>
```

Reply:
Thanks! You were too fast with the review :)